tools/logger/logger.c

```diff
@@ -270,7 +270,6 @@ int main(int argc, char *argv[])
 				usage();
 			}
 			config.dump_ldc = 1;
-			config.version_fw = 0;
```

Comment:
What motivation stands behind this change ?

Reply:
Thanks @ktrzcinx for reviewing this PR, really appreciated.

This is what allows the `-d ldc_file` option to ALSO show the checksum expected by the firmware - when and only when a firmware is running. See sample output in the description. I suggest looking at each commit independently if you have looked only at the entire diff for now.

Reply:
Ok, I've got it!